frontend/ui/data/koptoptions.lua

```diff
@@ -572,10 +572,6 @@ This can also be used to remove some gray background or to convert a grayscale o
                 toggle = {_("0 deg"), _("5 deg"), _("10 deg")},
                 values = {0, 5, 10},
                 default_value = DKOPTREADER_CONFIG_AUTO_STRAIGHTEN,
-                show = false, -- does not work (and slows rendering)
-                enabled_func = function(configurable)
-                    return optionsutil.enableIfEquals(configurable, "text_wrap", 1)
-                end,
```

Comment:
Removing stuff is fine, but you should add some bits to compensate :):
- a `help_text=`, describing what this does and what the button values means, and any other stuff we might have added to the bottom menu items and didn't care adding to this one (should it have a fine tuning button ?)
- should these buttons show `10°` instead of `10 deg` to ease/not need translations ?
- and obviously, you won't forget to add this new setting to Dispatcher :)

Reply:
+1 on 10°

Reply:
> Removing stuff is fine, but you should add some bits to compensate :):
> 
>     * a `help_text=`, describing what this does and what the button values means, and any other stuff we might have added to the bottom menu items and didn't care adding to this one (should it have a fine tuning button ?)

Done
>     * should these buttons show `10°` instead of `10 deg` to ease/not need translations ?
Done

>     * and obviously, you won't forget to add this new setting to Dispatcher :)
It's already there, just needed enabling.

Reply:
@yparitcher Forgot to push? :-)

Reply:
On the way, fixeing some other things together